includes/class-wc-stripe-api.php

```diff
@@ -363,15 +376,14 @@ public static function request_with_level3_data( $request, $api, $level3_data, $
 			set_transient( 'wc_stripe_level3_not_allowed', true, 3 * MONTH_IN_SECONDS );
 		} elseif ( $is_level_3data_incorrect ) {
 			// Log the issue so we could debug it.
-			WC_Stripe_Logger::log(
-				'Level3 data sum incorrect: ' . PHP_EOL
-				. print_r( $result->error->message, true ) . PHP_EOL
-				. print_r( 'Order line items: ', true ) . PHP_EOL
-				. print_r( $order->get_items(), true ) . PHP_EOL
-				. print_r( 'Order shipping amount: ', true ) . PHP_EOL
-				. print_r( $order->get_shipping_total(), true ) . PHP_EOL
-				. print_r( 'Order currency: ', true ) . PHP_EOL
-				. print_r( $order->get_currency(), true )
+			WC_Stripe_Logger::error(
+				'Level3 data sum incorrect',
+				[
+					'error'         => $result->error,
+					'order_line_items'      => $order->get_items(),
```

Comment:
Nit: the alignment here is slightly off.
```suggestion
					'error'                 => $result->error,
					'order_line_items'      => $order->get_items(),
```